workspaces/ui-v2/src/pages/diffs/ReviewEndpointDiffPage/ReviewEndpointDiffPage.tsx

```diff
@@ -112,6 +113,20 @@ export const ReviewEndpointDiffPage: FC<ReviewEndpointDiffPageProps> = ({
     }
   );
 
+  // We should redirect using useRedirectForDiffCompleted
+  if (allDiffs.length === 0) {
```

Comment:
Our diff interpretations can be filtered out (if there's invalid diffs, for a number of reasons in `ParsedDiff`) - however our code doesn't actually handle this. 

This functionally is still wrong, but doesn't crash the page. We should tackel this when we migrate the diff code to redux (such that we don't show invalid diffs at all to the UI - kinda hard now since we cannot tell until we intepret the diffs to see if they are invalid)